pkg/clusteragent/admission/mutate/autoinstrumentation/auto_instrumentation.go

```diff
@@ -290,35 +208,44 @@ func libImageName(registry string, lang language, tag string) string {
 	return fmt.Sprintf(imageFormat, registry, lang, tag)
 }
 
+func (w *Webhook) isPodEligible(pod *corev1.Pod) bool {
+	return ShouldInject(pod)
+}
+
+// isEnabledInNamespace indicates if Single Step Instrumentation is enabled for
+// the namespace in the cluster
+func (w *Webhook) isEnabledInNamespace(namespace string) bool {
```

Comment:
Similarly `isEnabledInNamespace` just calls the global `IsEnabledInNamespace` so that other packages don't need to depend on the webhook (prme-4).